### DISCUSSION TITLE
Nothing happens

### DISCUSSION THREAD

**Discussion by ewswlw** [DaxStudio/DaxStudio#998] [Q&A] [answered]
New to using DAX studio. The log shows that my power pivot model is connected, and i see my tables/columns on the left....but when i actually try to run a query (for example.......EVALUATE Table1....then RUN) nothing happens its just blank and does not return anything

**Comment by dgosbell** [comment, marked as answer]
When you click on the run button are you clicking on the top half of the button which triggers the action or are you clicking on the bottom half of the button which just changes the mode?
![image](https://user-images.githubusercontent.com/8241830/204162556-a9edbbcd-4234-49cc-8800-95411994d881.png)
You need to click on the top half of the split button to actually trigger the action.

**Reply by ewswlw** [reply]
Works now. Thank you very much
